go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/alecthomas/kong v0.8.0
 	github.com/bufbuild/buf v1.25.1
-	github.com/crossplane/crossplane-runtime v0.20.0
+	github.com/crossplane/crossplane-runtime v0.20.1
```

Comment:
This was already bumped by Renovate, not sure why it didn't bump the `k8s.io/*` dependencies too 🤔